### DISCUSSION TITLE
Unable to start the Python language server.

### DISCUSSION THREAD

**Discussion by widkit** [atom/atom#22613] [General]
Unable to start the Python language server. Make sure to install pylsp 0.19 or newer by running: python -m pip install ‘python-lsp-server[all]’ python -m pip install git+https://github.com/tomv564/pyls-mypy.git So I’m using Kubuntu 21.04 and I’m messing with this error over an hour. What do I do? I got pylsp and python both installed. I tried reinstalling from deb instead of snap package, didn’t fixed it.

**Comment by ThatXliner** [comment]
@aminya can help. Also, this should be asked in the ide-python package

**Comment by aminya** [comment]
@widkit Check this solution and see if it fixes it
https://github.com/atom-community/ide-python/issues/182#issuecomment-805799708

Please continue the discussion there.
